react/week3/todo-app/src/Components/Todo.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from "react";
+import Border from "./Border";
+import PropTypes from "prop-types";
+function Todo({ toDo, deleteTodo, updateTodo }) {
+  const [status, setStatus] = useState(false);
+  const [isEditing, setEditState] = useState(false);
+  const [newdescription, setDescription] = useState(toDo.description);
+
+  const check = () => {
+    setStatus(!status);
+  };
+
+  const editStatus = () => {
+    return setEditState(!isEditing);
+  };
+
+  const updateButton = () => {
+    if (newdescription === "") {
+      alert("plese enter the task");
+    } else {
+      updateTodo(toDo.id, newdescription);
+      setEditState(!isEditing);
+    }
+  };
+  const editingTemplate = (
+    <>
+      <li className="todo">
+        <input
+          value={newdescription}
+          type="text"
+          onChange={(event) => {
+            setDescription(event.target.value);
+          }}
+        />
+        <button className="update-button" onClick={updateButton}>
+          update
+        </button>
+        <button className="delete-button" onClick={() => deleteTodo(toDo.id)}>
+          delete
+        </button>
+      </li>
+    </>
+  );
+
+  const viewTemplate = (
+    <>
```

Comment:
```suggestion
    <li>
```
only li can be a child element of ul, please check HTML structure